### ISSUE TITLE
v1.0.1 Breaks When Running on Travis

### ISSUE THREAD

**Issue by stdavis** [tholo/pytest-flake8#48]
[My Travis build](https://travis-ci.org/agrc/honeycomb/builds/393862705) broke when it upgraded this project from v1.0.0 to v1.0.1. [Downgrading to v1.0.0](https://github.com/agrc/honeycomb/commit/6539fd903ae93ab0f4944758ffe43d50049500fa) [fixed my build](https://travis-ci.org/agrc/honeycomb/builds/393871680).

Here's [the error](https://travis-ci.org/agrc/honeycomb/builds/393862705#L653) that I was receiving:
```
The command "pytest -s --cov=honeycomb --flake8" exited with 1.
```

**Comment by revmischa** [comment]
I am also getting error `pytest: error: unrecognized arguments: --flake8` on travis, with python 3.7

**Comment by tholo** [comment]
I *think* this got fixed with the changes made in 1.0.3...
